src/components/application_manager/src/policies/policy_handler.cc

```diff
@@ -809,6 +862,12 @@ std::string PolicyHandler::OnCurrentDeviceIdUpdateRequired(
   return device_params.device_mac_address;
 }
 
+std::map<std::string, std::string> PolicyHandler::GetRegisteredLinks() const {
```

Comment:
@okozlovlux why do you need both functions GetRegisteredLinks(map) and GetRegisteredLinks()? Please left only one to avoid confusing calls and multiple usage of both methods